Source/RestKit/RestRequest.swift

```diff
@@ -303,7 +303,36 @@ public struct RestRequest {
             completionHandler(dataResponse)
         }
     }
-    
+
+    public func responseVoid(
```

Comment:
I see your point, but I don't know that there is a better alternative.  I think adding documentation as you recommend in your previous comment is best solution here.